apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -427,10 +428,11 @@ class ReviewTab extends Component {
           <div style={styles.reviewHeader}>
             {viewAs !== ViewType.Teacher &&
               !errorLoadingReviewblePeers &&
+              codeReviewEnabled &&
               (viewAsCodeReviewer
                 ? this.renderBackToMyProject(this.onClickBackToProject)
                 : this.renderPeerDropdown(reviewablePeers, this.onSelectPeer))}
-            {this.renderReadyForReviewCheckbox()}
+            {codeReviewEnabled && this.renderReadyForReviewCheckbox()}
```

Comment:
nit: the logic for showing/hiding the checkbox all resides in `renderReadyForReviewCheckbox` so could we read the `codeReviewEnabled` flag there? FWIW this component could definitely use some logic cleanup down the road, so I imagine this will hopefully get simplified soon :)